backend/btrixcloud/basecrawls.py

```diff
@@ -62,8 +67,12 @@ def __init__(
         self.colls = colls
         self.storage_ops = storage_ops
 
-        self.presign_duration_seconds = (
-            int(os.environ.get("PRESIGN_DURATION_MINUTES", 60)) * 60
+        presign_duration_seconds = (
+            int(os.environ.get("PRESIGN_DURATION_MINUTES", PRESIGN_MINUTES_DEFAULT))
```

Comment:
We also have:
```
  PRESIGN_DURATION_MINUTES: "{{ .Values.storage_presign_duration_minutes | default 60 }}"
```
in configmap.yaml so that will take precedence. probably should keep this default in one place only to avoid confusion, can make the env var blank by default

Reply:
oh, I see its been updated to latest value, but should it be blank there perhaps?